docs/toc.md

```diff
@@ -1,46 +1,56 @@
-# [Programming languages on the .NET Platform](languages/index.md)
-## [🔧 .NET Platform & programming languages](languages/overview.md)
-## [🔧 C# programming language](languages/c-sharp.md)
-## [🔧 F# programming language](languages/f-sharp.md)
-## [🔧 VisualBasic.NET programming language](languages/visual-basic.md)
-## [🔧 CLI programming languages](languages/cli-languages.md)
-# [Base Class Library](bcl/index.md)
-## [🔧 Base Class Library Overview](bcl/overview.md)
-## [🔧 BCL API reference documentation](bcl/api-ref.md)
-# [Asynchronous Programming with .NET](async/async-index.md)
-## [Async Overview](async/async-overview.md)
-## [Async Programming in C#/VB](async/async-csharp-vb.md)
-## [Async Programming in F#](async/async-fsharp.md)
-# [Advanced topics](advanced/index.md)
-## [.NET Compiler Platform (“Roslyn”)](advanced/roslyn/index.md)
-### [.NET Compiler Platform (“Roslyn”) Overview](advanced/roslyn/roslyn-overview.md)
+## [🔧 What is the .NET Platform?](getting-started/what-is-dotnet.md)
+## [Installing .NET Core](getting-started/installing/index.md)
+### [Installing .NET Core on Windows](getting-started/installing/installing-core-windows.md)
+### [Installing .NET Core on Linux](getting-started/installing/installing-core-linux.md)
+### [Installing .NET Core on OS X](getting-started/installing/installing-core-osx.md)
+## [Setting up your development environment](getting-started/devenv/index.md)
+### [🔧 Development using Visual Studio 2015](getting-started/devenv/using-visual-studio.md)
+### [🔧 Development using Visual Studio Code](getting-started/devenv/using-visual-studio-code.md)
+### [🔧 Development using vim](getting-started/devenv/using-vim.md)
+### [🔧 Development using Sublime Text](getting-started/devenv/using-sublime.md)
+## [Writing your first console application on .NET Core](getting-started/first-console-app.md)
+# [Learn C#](languages/csharp/index.md)
+## [🔧 Syntax](languages/csharp/syntax.md)
+## [🔧 Features at a glance](languages/csharp/features.md)
+## [🔧 C# Type system](languages/csharp/type-system.md)
+## [🔧 Iterators](languages/csharp/iterators.md)
+## [🔧 Delegates & events](languages/csharp/delegates-events.md)
+## [🔧 Generics](languages/csharp/generics.md)
+## [🔧 Language Integrated Query (LINQ)](languages/csharp/linq.md)
+## [🔧 Asynchronous programming](languages/csharp/async.md)
+## [🔧 Paralell programming](languages/csharp/parallel.md)
+## [🔧 Native interoperability](languages/csharp/interop.md)
+## [🔧 Reflection & code generation](languages/csharp/reflection.md)
+## [🔧 Documenting your code](languages/csharp/codedoc.md)
+# [Understanding the .NET Core Platform](index.md)
+## [The .NET Primer](concepts/primer.md)
+## [Programming Languages on .NET Platform](languages/index.md)
+### [🔧 .NET - a multi-language platform](languages/overview.md)
+### [🔧 The main trio: C#, F# and VB.NET](languages/main-trio.md)
+## [Writing console applications](console/index.md)
+### [ Overview and moving pieces](console/overview.md)
```

Comment:
Per my earlier comment - "The main trio - C#, F# and VB.NET" should be "C#, F# and Visual Basic".


Otherwise LGTM.


Sent from Outlook<http://aka.ms/weboutlook>

________________________________
From: Phillip Carter <notifications@github.com>
Sent: Monday, November 30, 2015 4:11:09 PM
To: dotnet/core-docs
Cc: Steve Hoag
Subject: Re: [core-docs] Update TOC and directory structure (#175)


In docs/toc.md<https://github.com/dotnet/core-docs/pull/175#discussion_r46224025>:

> +## [? Iterators](languages/csharp/iterators.md)
> +## [? Delegates & events](languages/csharp/delegates-events.md)
> +## [? Generics](languages/csharp/generics.md)
> +## [? Language Integrated Query (LINQ)](languages/csharp/linq.md)
> +## [? Asynchronous programming](languages/csharp/async.md)
> +## [? Paralell programming](languages/csharp/parallel.md)
> +## [? Native interoperability](languages/csharp/interop.md)
> +## [? Reflection & code generation](languages/csharp/reflection.md)
> +## [? Documenting your code](languages/csharp/codedoc.md)
> +# [Understanding the .NET Core Platform](index.md)
> +## [The .NET Primer](concepts/primer.md)
> +## [Programming Languages on .NET Platform](languages/index.md)
> +### [? .NET - a multi-language platform](languages/overview.md)
> +### [? The main trio: C#, F# and VB.NET](languages/main-trio.md)
> +## [Writing console applications](console/index.md)
> +### [ Overview and moving pieces](console/overview.md)


Yeah, looking at it again and I think "moving pieces" fits with the overall TOC tone a bit better.

-
Reply to this email directly or view it on GitHub<https://github.com/dotnet/core-docs/pull/175/files#r46224025>.